script.js

```diff
@@ -1,4 +1,73 @@
+var playerOneGuess = "";
+var playerTwoGuess = "";
+var diceRollOne = 0;
+var diceRollTwo = 0;
+var playerOneNumber = 0;
+var playerTwoNumber = 0;
+var gameState = "";
+var myOutputValue = "";
+
+//dice roll function
+var diceRoll = function () {
+  var oneToFive = Math.floor(Math.random() * 6);
+  var randomNumber = oneToFive + 1;
+  return randomNumber;
+};
+//Helper function for selecting combination which gives largest value
+var playerOneCombinationGenerator = function () {
+  diceRollOne = diceRoll();
+  diceRollTwo = diceRoll();
+  if (diceRollOne >= diceRollTwo) {
+    var playerOneGuess = `${diceRollOne}${diceRollTwo}`;
+    playerOneNumber = Number(playerOneGuess);
+    myOutputValue = `Welcome Player One! You have rolled ${diceRollOne} for Dice One and ${diceRollTwo} for Dice Two. <br> The largest possible combination is ${playerOneGuess}.It is now Player 2\'s turn.`;
+  }
+  if (diceRollTwo > diceRollOne) {
+    var playerOneGuess = `${diceRollTwo}${diceRollOne}`;
+    playerOneNumber = Number(playerOneGuess);
+    myOutputValue = `Welcome Player One! <br> You have rolled ${diceRollOne} for Dice One and ${diceRollTwo} for Dice Two. <br> The largest possible combination is ${playerOneGuess}.It is now Player 2\'s turn.`;
```

Comment:
We typically use another variable instead of myOutputValue in the helper functions, so that we can use myOutputValue in the main function

Reply:
Ah ok...so for the helper function I should just have ended it at line 27 to generate the highest possible number?

Reply:
Nope the code is correct. The point is that different functions should not be returning the same variable as this can lead to errors when the computer is reading the programme. 

So try to use and return different variables in each function instead of myOutputValue at lines 30, 46, 72.
We don't typically make myOutputValue a global variable so if you need to then something isn't quite right